src/logic/wallets/getWeb3.ts

```diff
@@ -114,7 +115,7 @@ export const reverseENSLookup = async (address: string): Promise<string> => {
     return ''
   }
 
-  return verifiedAddress === address ? name : ''
+  return verifiedAddress === checksumAddress(address) ? name : ''
```

Comment:
How does this line fix the issue? I see that is working but...

Reply:
`verifiedAddress` is returned already checksummed. The comparison was failing because `address` may or may not be checksummed already.

Reply:
Oh I see now... Isn't there a string comparison ignoring the checksum address?